### ISSUE TITLE
Homework 2

### ISSUE THREAD

**Issue by aaronkaplan** [OreoCassowary/datascience-workshop#4]
# Use the Berkley weather data to produce the following CSV output file:

```
time ; weather_station_id; lat; lon; temperature(Celsius); Uncertainty
```

## detailed format description
  * time ..... please use YYYY-MM-DD format . If there are multiple measurements on that day for that specific station, take the first one
  * weather_station_id .... int
  * lat/lon .... float (use ".", not ",")
  * temperature .... float (average temperature for time frame)
  * uncertainty.... float

Use the following data as input data: http://berkeleyearth.org/source-files/ Especially: http://berkeleyearth.lbl.gov/downloads/Sources/World%20Weather%20Records/TAVG/LATEST.zip

After you combined the weather_station_id and lat and long (hint: read station_id and lat, lon *first* and put it into a dict), parse the whole ``data.txt`` file , **join** the two files (via the dict) and print the result as a CSV file.
Think what you will do if a station_id is in the ``data.txt`` file but not in the ``site_summary.txt`` (or ``site_detail.txt``)

Do write a README.md file to document your program and your thoughts.

Do proper checks! Ask yourself a question such as , what is the average temperature in Antarctica. Google for it, compare data and find rows where it does not match for example.
In other words: test your assumptions! Document your checks.

# questions about the data:
* how many measurements / station => plotly (station color indicates measurement freq)
[measurement frequency](https://plot.ly/~MillivoltLightningbolt/1/)

* worldwide avg temp / year => plotly timeseries
* worldwide avg temp / month => plotly timeseries
* nr of stations / country (just for fun)

**Comment by artkapl** [comment]
Note: dict = {station_id : [lat, lon]}

**Comment by artkapl** [comment]
Learn how to write clean function docstrings.

- [Wikipedia: Docstring](https://en.wikipedia.org/wiki/Docstring)  
- [Stackoverflow: standard Python docstring format](https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format)
